src/migrations/1657204514370-canvas-visual.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+const table = '`canvas`';
+const column = '`bannerCardID`';
+
+export class canvasVisual1657204514370 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE ${table} ADD ${column} char(36) NOT NULL`
```

Comment:
You defined `bannerCard` as optional in entity and interface, adding NOT NULL in migration will basically make it mandatory replacing NULL values with empty strings.